File: C Modules/Assignments/Assignment no 6/Question-2/Assignment no 3/Type-2/Question_4.c
#include<stdio.h>
void prime(int n)
{
    int i=2;
    int flag=1;
    if (n<=1)
    {
        printf("It is not prime number:");
    }
    while (i<=n/2)
    {
        if (n%i==0)
        {
            flag==0;
            break;
        }
        i++;
    }
    if (flag==1)
    {
        printf("%d is prime number",n);
    }
    else{
        printf("%d is not a prime number",n);
    }
}
int main()
{
    int n,i=2,flag=1;
    printf("enter the number:");
    scanf(" %d", &n);
    prime(n);
}

File: Java Module/Assignments in c to java/Assignment-04/Question_4.java
//Print strong numbers in the given range 1 to n.
public class Question_4 {

    public static void main(String[] args) {
            int n=10;
    // printf("Enter the number\n");
    // scanf(" %d", &n);
    
   System.out.printf("strong numbers in the given range 1 to %d :\n",n);

    for (int num=1; num<=n;num++)
    {
        int temp=num;
        int sum=0;
        while (temp>0)
        {
            int digit=temp%10;
            int fact=1;
        for (int i = 1; i<=digit; i++)
        {
            fact=fact*i;
        }
        sum=sum+fact;
        temp=temp/10;
        }

    if (sum==num)
    {
        System.out.printf("strong number: %d\n",num);
    }
    else{
        System.out.printf("not a strong number: %d\n",num);
    }   
}
    }
}